pkg/store/gcworker/gc_worker_test.go

```diff
@@ -1703,15 +1671,13 @@ func TestResolveLocksPhysical(t *testing.T) {
 
```

Comment:
The test `TestResolveLocksPhysical` which is litteraly a test specific to Green GC can be deleted